app/Helpers/HasTags.php

```diff
@@ -8,7 +8,7 @@
 trait HasTags
 {
     /**
-     * @return \App\Models\Tag[]
+     * @return \Illuminate\Database\Eloquent\Collection
```

Comment:
Similar docblocks have been changed in this PR because we're using these methods as collections elsewhere in the code. For instance, `$article->tags->pluck()`.